.pre-commit-config.yaml

```diff
@@ -11,7 +11,7 @@ repos:
       - id: no-commit-to-branch
         args: [--branch, develop, --branch, main]
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: "v0.1.6"
+    rev: "v0.1.7"
```

Comment:
Running `poetry update` also updated the version of ruff we use. Without this change the `test_pre_commit_versions_are_in_sync` test was failing